# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/mochitests/file_testloadflags.js
# Commit: 39f2e21623aa
# Full Hash: 39f2e21623aa7c19517e32cbd97dc1c3584c58ca
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-02 03:24:46
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r 2c873c72bf1f -r 39f2e21623aa netwerk/test/mochitests/file_testloadflags.js
--- a/netwerk/test/mochitests/file_testloadflags.js	Mon Jun 01 16:49:05 2020 +0000
+++ b/netwerk/test/mochitests/file_testloadflags.js	Mon Jun 01 16:49:03 2020 +0000
@@ -36,7 +36,12 @@
 
   var prefSet = new Promise(resolve => {
     SpecialPowers.pushPrefEnv(
-      { set: [["network.cookie.cookieBehavior", 1]] },
+      {
+        set: [
+          ["network.cookie.cookieBehavior", 1],
+          ["network.cookie.sameSite.schemeful", false],
+        ],
+      },
       resolve
     );
   });